packages/demo/src/App.tsx

```diff
@@ -1,10 +1,16 @@
-import { Highlight, themes } from "prism-react-renderer"
+import { Highlight, Prism, themes } from "prism-react-renderer"
 import styles from "./app.module.css"
 import clsx from "clsx"
 import { ProjectBadge } from "formidable-oss-badges"
 import { useState } from "react"
 import { sampleCode } from "./sample-code"
 
+// Example of importing a custom language directly from Prism
+typeof window !== "undefined" && (window.Prism = Prism)
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+await import("prismjs/components/prism-applescript")
+
```

Comment:
👇🏻 example of AppleScript highlighting. As a note TypeScript doesn't love importing languages from `prismjs/components` so we need to `@ts-ignore` it, despite the exports existing.

<img width="1244" alt="Screenshot 2023-04-21 at 8 07 37 AM" src="https://user-images.githubusercontent.com/1738349/233643843-82a4a634-9ee2-4667-8baa-a4c922191238.png">